src/clang-cpp-frontend/clang_cpp_convert.cpp

```diff
@@ -703,7 +703,7 @@ void clang_cpp_convertert::build_member_from_component(
   // Add this pointer as first argument
   std::size_t address = reinterpret_cast<std::size_t>(fd.getFirstDecl());
 
-  this_mapt::iterator it;
+  this_mapt::iterator it = this_map.find(address);
```

Comment:
I still think we should get rid of this map.

Reply:
ACK